### PR TITLE
Index title fix

### PR DESCRIPTION
**Pull request by emmaohn**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Levi's website | Home</title>
```
Comment (levi-wj) [inline src/index.html:6]: I thought you were a native english speaker
fix  it nerd

**Review by levi-wj — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
...
+    <title>Sleep outside | Home</title>
```
Comment (levi-wj) [inline src/index.html]: I thought you were a native english speaker
fix it nerd

Comment (emmaohn) [reply src/index.html]: I am the best English speaker you know

**Review by levi-wj — APPROVED** [review]
good job hotsexy

**Pull request merged**